.github/workflows/build.yml

```diff
@@ -260,3 +264,141 @@ jobs:
         with:
           name: Windows-PerformanceTestResults.txt
           path: ./out/SoarPerformanceTests/PerformanceTestResults.txt
+
+  python_wheels:
+    name: wheel-*nix
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [
+            ubuntu-latest,
+            # latest available X86_64 target
+            macos-12,
+            # latest is ARM
+            macos-latest,
+
+            windows-latest,
+          ]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          # Do a complete clone, not a shallow one.
+          # https://github.com/actions/checkout/#usage
+          fetch-depth: 0
```

Comment:
Why is a complete clone required?

Reply:
Because else `git describe` does not properly catch the tag that the current commit is based on, or find the latest tag, that it requires to calculate the version tag.

Should I add a comment about this?

Reply:
Ah, right! Yes, a comment would be good.